Makefile

```diff
@@ -527,6 +529,33 @@ cos-fleet-catalog-camel/teardown:
 	rm config/connector-types/cos-fleet-catalog-camel
 .PHONY: cos-fleet-catalog-camel/teardown
 
+# Touch all the necessary files for fleet manager to start
+# See docs/populating-configuration.md for more information
+secrets/setup/empty:
```

Comment:
Nice. I am wondering if we can document somewhere (e.g https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/tree/aa91c8702ad01daa7da5840f792de435bcd04bef/docs/best-practices) that this list needs to be kept up to date

Reply:
I will think about this and maybe add some information if I end up finding it appropriate in a potential future PR.
At the moment I don't have a clear idea or where this would be or if would even add any substantial as the same thinking can be applied to lots of tasks related to Makefile/templates

Reply:
Okay sounds good to me. 

PR template could be another place to add it as well - but I am happy to do it once we are clear of the place we want to add it.